runtime/missingmember_test.go

```diff
@@ -4163,3 +4163,837 @@ transaction(recipient: Address, amount: UFix64) {
 	)
 	require.NoError(t, err)
 }
+
+func TestRuntimeMissingMemberExampleMarketplace(t *testing.T) {
+
+	runtime := newTestInterpreterRuntime()
+
+	exampleTokenAddress, err := common.HexToAddress("0x1")
+	require.NoError(t, err)
+
+	exampleNFTAddress, err := common.HexToAddress("0x2")
+	require.NoError(t, err)
+
+	exampleMarketplaceAddress, err := common.HexToAddress("0x3")
+	require.NoError(t, err)
+
+	const exampleTokenContract = `
```

Comment:
We already have some example and "real" contracts in the codebase, maybe see if it is possible to use them instead. See `examples.go`, `ft_test.go`, and `nft_test.go`

Reply:
Unfortunately it looks like the marketplace playground example that this test is based on has more complexity than the tokens in our examples, but less complexity than a real contract, so I don't think any of the pre-existing token contracts in our tests are going to work.